conda_smithy/configure_feedstock.py

```diff
@@ -2230,6 +2239,7 @@ def render_pixi(jinja_env, forge_config, forge_dir):
     new_file_contents = template.render(
         smithy_version=__version__,
         platforms=sorted(platforms),
+        all_shellcheck_platforms=CONDA_FORGE_SHELLCHECK_PLATFORMS,
```

Comment:
I guess an alternative would be something more like `CONDA_FORGE_TOOL_PLATFORMS = {"shellcheck": ["..."]}`